codes/quantum/oscillators/cat_code.yml

```diff
@@ -0,0 +1,97 @@
+
+# This field starts a list of specific labeled subfields; do not leave it
+# empty. If empty, comment out. Also, indentations are important!
+features:
+
+  # The fields below relate to specific properties of the code that are worth
+  # noting. In some cases (see below), there might be multiple pieces of
+  # information associated with a field (e.g., mutliple encoders/decoders,
+  # multiple facts about fault tolerance, etc.); in such cases, the field
+  # accepts a list of text entries, with one item per piece of information.
+  #
+  # Any fields below should be omitted if they do not apply.
+
+  rate: 'Encode a 2-dimensional logical Hilbert space into an infinite dimensional Hilbert space. A cat code with \(Z_{2(S+1)}\) rotation symmetry can protect against \(S\)-photon loss/gain errors and phase space rotation error up to \(\frac{\pi}{2(S+1)}\).'
+
+  # This field that starts a list of strings, indexed by a dash; do not leave it
+  # empty. If empty, comment out.
+  encoders: 
+    - 'Project a coherent state onto a subspace of occupation number modulo \(2(S+1)\). It is a superposition of coherent states around a circle.'
+    
+
+  #transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  #general_gates:
+    #- ''
+
+  decoders:
+    - 'Measurement in the Fock basis. For a \(2(S+1)\) cat code, a number measurement returns outcome \(2(S+1)k\), if \(k\) is even, then it corresponds to logical 0 state; if \(k\) is odd, then it corresponds to logical 1 state.' 
+    #- '' 
+
+  #fault_tolerance:
+    #- 'Noise-model-preserving gadgets, noise-biased gates, fault-tolerant flag error correction'
+    #- 'Pieceable fault tolerance.'
+
+  #threshold: 
+    #- 'Threshold standard deviation of displacement errors is 11.2dB if Gaussian displacement error only applies to data and ancillas \cite{arxiv:another-paper}.'
+    #- '0.3\% error-correction threshold ... with noisy MWPM decoder \cite{doi:good-paper}.'
+
+  
+  # Propose a field that you feel is applicable to a large and interesting class of codes
+  #some_other_field:
+  # - 'detail'
+
+realizations:
+  # List and explain the different "domains" of realizations in list items.
+  # Group closely related realizations into a single item. It's OK to have a
+  # single list item. Each item should more or less correspond to what would be
+  # said in a paragraph.
+  - '\(S=1\) cat state has been realized in superconducting circuit \cite{doi:10.1038/nature18949}.'
+
+#notes:
+  #- 'Can be used to prepare biased error correcting code by choosing \(\alpha\neq \beta\).'
+  #- 'Can be used to encode the physical qubits for surface code.'
+  #- 'GKP states are non-Gaussian states.'
+  #- 'Can reference other codes, but try to put such connections in the relations fields.'
+  #- 'Anything noteworthy that''s not above or below.'
+
+relations:
+
+  cousins:
+    - code_id: gkp
```

Comment:
This is a nice idea for a cousin, but its applicable to a larger class of codes. I've moved it to bosonic_rotation codes to be submitted by jtiosue.